### DISCUSSION TITLE
Error joining Slack

### DISCUSSION THREAD

**Discussion by ajkourabi** [OpenMined/courses#345] [General]
Hello, 
I am unable to join the slack using the link from the website, it seems it has become unavailable. Would anyone be able to send me an invite? 
![image](https://user-images.githubusercontent.com/63927131/123875524-d6ff8300-d907-11eb-90a5-3e486128d1b2.png)

**Comment by kerinb** [comment]
I am getting a similar issue; each email address I try, I get 'token_revoked' error? Does anyone know way around?
